cnf-certification-test/lifecycle/suite.go

```diff
@@ -483,3 +490,17 @@ func testPodPersistentVolumeReclaimPolicy(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+func testContainersImageTag(env *provider.TestEnvironment) {
+	badcontainers := []string{}
+	for _, cut := range env.Containers {
+		logrus.Debugln("check container ", cut.String(), " image, should be tagged ")
+		if cut.ContainerImageIdentifier.Tag == "" {
+			badcontainers = append(badcontainers, "{"+cut.String()+": is not having tag"+"}")
```

Comment:
That's a strange format. I'd stick to just adding the cut.String() to the slice. After all, when printing the log messages you are already stating that they refer to containers not having their images tagged.